README.md

```diff
@@ -29,9 +29,11 @@ To manage Pub/Sub resources associated with different Projects, you may use diff
 
```

Comment:
https://github.com/electricimp/Google-PubSub/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R26 - (line 26) typo, update to:
Google Cloud Project is a basic entity of Google Cloud Platform which allows users to create...